manage_breast_screening/notifications/models.py

```diff
@@ -143,6 +144,7 @@ class Meta:
     completed_at = models.DateTimeField(null=True)
     attended_not_screened = models.CharField(max_length=30, default="")
     assessment = models.BooleanField(default=False)
+    originating_file = ArrayField(models.CharField(max_length=50), default=list)
```

Comment:
`originating_file` is probably not what we want to call this field as it can contain multiple values and _originating_ feels like it relates to record creation where one or more of the values could relate to updating the record.

Something like `extract_files` maybe. It should be plural whatever we choose.